ispdb/email.it.xml

```diff
@@ -5,29 +5,26 @@
     <displayName>email.it</displayName>
     <displayShortName>email.it</displayShortName>
     <incomingServer type="imap">
-      <hostname>imapmail.email.it</hostname>
+      <hostname>in.email.it</hostname>
       <port>993</port>
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
-      <authentication>password-encrypted</authentication>
+      <authentication>password-cleartext</authentication>
     </incomingServer>
     <incomingServer type="pop3">
-      <hostname>popmail.email.it</hostname>
+      <hostname>in.email.it</hostname>
```

Comment:
The [FAQ](https://www.email.it/faq) states that the host name for POP3 is `pop.email.it`. Looking at the DNS record, it seems to point to the exact same place as `in.email.it` (which seems confirmed by your openssl testing), so using `in.` should not break anything, but I'd prefer if we used the same subdomain as mentioned in the FAQ to avoid confusing users.

Reply:
Nice catch :+1: